README.md

```diff
@@ -1,17 +1,32 @@
-# SpatialOS Unreal GDK Starter Project
+# SpatialOS GDK for Unreal Starter Project
 
-## Introduction
+* **LICENSE:** Use of the contents of this repository is subject to the [Improbable license](LICENSE.md)
+* **REPOSITORY:** github.com/spatialos/UnrealGDKStarterProject
 
-You can use the Starter Project as the basis for developing a game using the SpatialOS GDK for Unreal Engine. To use the Starter Project, you need to follow the [setup guide](https://github.com/improbable/UnrealGDK/blob/master/docs/setup-and-installing.md) for the GDK. We've enabled use of the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
+This Starter Project is both a reference project and a starting point for building games on SpatialOS and Unreal Engine 4.  The project uses the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
 
-> Note that the SpatialOS GDK for Unreal Engine is currently in **alpha**, with stability and performance improvements pending.
+For setup instructions, follow the [Get started](https://docs.improbable.io/unreal/latest/get-started/introduction) guide in the GDK for Unreal documentation.
```

Comment:
Should this link to https://docs.improbable.io/unreal/pre-alpha/setup-and-installing? (Current link is dead)

Also, how come this is an absolute link rather than relative? Is it to do with getting the docs from GitHub onto the main docs site?

Reply:
This is correct once we have finished the getting started docs